AUDIO/HintAmbientAudioBank.md

```diff
@@ -5,15 +5,13 @@ ns: AUDIO
 
 ```c
 // 0x8F8C0E370AE62F5C 0xF1850DDC
-Any HINT_AMBIENT_AUDIO_BANK(Any p0, Any p1);
+cs_type(Any) BOOL HINT_AMBIENT_AUDIO_BANK(cs_type(Any) char* bankName, cs_type(Any) bool bOverNetwork);
```

Comment:
This native has 3 param actually.
- The bank name
- If network
- Player bits (int)

Reply:
This doesn't matter, this native shouldn't be used, and you can't add additional arguments to a native doc without breaking compatibility.